Assets/LeapMotion/Core/Scripts/Utils/RunTimeOnly.cs

```diff
@@ -0,0 +1,24 @@
+﻿/******************************************************************************
+ * Copyright (C) Leap Motion, Inc. 2011-2017.                                 *
+ * Leap Motion proprietary and  confidential.                                 *
+ *                                                                            *
+ * Use subject to the terms of the Leap Motion SDK Agreement available at     *
+ * https://developer.leapmotion.com/sdk_agreement, or another agreement       *
+ * between Leap Motion and you, your company or other organization.           *
+ ******************************************************************************/
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Leap.Unity.Attributes {
+
+  public class RunTimeOnly : CombinablePropertyAttribute, IPropertyDisabler {
```

Comment:
It's standard convention to suffix attributes with the word 'Attribute', so to make it `RunTimeOnlyAttribute`.  It still lets you type `[RunTimeOnly]` when you use it though.